pkgs/applications/editors/emacs/macport.nix

```diff
@@ -114,7 +113,7 @@ stdenv.mkDerivation rec {
       extensions are distributed with GNU Emacs; others are available
       separately.
 
-      This is the "Mac port" addition to GNU Emacs 26. This provides a native
+      This is the "Mac port" addition to GNU Emacs. This provides a native
```

Comment:
This was confusing to me, since it's not currently anything to do with Emacs 26.